server/apps/immich/src/api-v1/auth/dto/login-credential.dto.ts

```diff
@@ -1,13 +1,14 @@
 import { ApiProperty } from '@nestjs/swagger';
 import { Transform } from 'class-transformer';
-import { IsNotEmpty } from 'class-validator';
+import { IsEmail, IsNotEmpty, IsString } from 'class-validator';
 
 export class LoginCredentialDto {
-  @IsNotEmpty()
```

Comment:
Should we still need to keep `@IsNotEmpty()`?

Reply:
Nah, an empty during doesn't pass as an email.

I test this case in the create user spec.